### PR TITLE
Removed unneccessary List unpacking

### LINKED ISSUE #800: Unnecessary List Unpacking
In the code snippet: https://github.com/mozilla/OpenWPM/blob/512018f852a2b554b5b08a72936f835adf6b7110/test/test_webdriver_utils.py#L29-L33 
List unpacking is very unnecessary here, I am not sure why `_` is used here, and what the use of it is.

Explanation:
Now, when I tried to debug what the `SELECT` query is returning is a list of two tuples, Now, to my understanding  one is the tuples is for `<class 'openwpm.Commands.Types.InitializeCommand'>` for initialization command and the second one is for `<class 'openwpm.Commands.Types.GetCommand'>`. 
To my understanding, for this test, we are only interested in `command_status` and `error` for `<class 'openwpm.Commands.Types.GetCommand'>`.

Potential fixes:
1. Remove the underscore and assign the result of SELECT command into get_command and in the assertion we can use appropriate indices. Something like:
`assert get_command[1][0] == "neterror"`
`assert get_command[1][1] == "dnsNotFound"`
2. Remove the underscore and assign indicised result to `get_command`. Something like:
```python
get_command = db_utils.query_db(
            manager_params["db"],
            "SELECT command_status, error " "FROM crawl_history ",
            as_tuple=True,
        )[1]
```

### PR DESCRIPTION
**Pull request by LordReigns**
Fixes #800

### CODE REVIEW

**Review by vringar — APPROVED** [review]

File: test/test_webdriver_utils.py [hunk 1]
```diff
@@ -26,10 +26,11 @@ def test_parse_neterror_integration(self):
         manager.close()
         # I couldn't figure out how to just get a the get_command
```
Comment (ankushduacodes) [inline test/test_webdriver_utils.py:27]: @vringar Is this comment needed anymore?

File: test/test_webdriver_utils.py [hunk 2]
```diff
@@ -26,10 +26,11 @@ def test_parse_neterror_integration(self):
...
         # Because the type has too many quotes
```
Comment (ankushduacodes) [inline test/test_webdriver_utils.py:28]: This one too

Comment (vringar) [reply test/test_webdriver_utils.py:28]: No, the two lines should be removed.
Btw you can do multiline comments by clicking and dragging on the left side of the screen.

Comment (ankushduacodes) [reply test/test_webdriver_utils.py:28]: Ah... Didn't know that😅, good to know.

Comment (ankushduacodes) [inline test/test_webdriver_utils.py:28]: @LordReigns Please remove this comment. Rest looks good.

**Review by ankushduacodes — CHANGES_REQUESTED** [review]

File: test/test_webdriver_utils.py (hunk 2 above)
Comment (LordReigns) [reply test/test_webdriver_utils.py:28]: Oh k. I will remove it

**Review by ankushduacodes — APPROVED** [review]

**Comment by LordReigns** [comment]
I pushed a commit with the comments removed @ankushduacodes

**Review by vringar — APPROVED** [review]

**Comment by LordReigns** [comment]
It seems the build fails. @vringar @ankushduacodes. An index out of range error. Do you know how that happens. It passed the local test cases.

**Comment by ankushduacodes** [comment]
> It seems the build fails. @vringar @ankushduacodes. An index out of range error. Do you know how that happens. It passed the local test cases.

Could you please post here the whole error traceback and elaborate on what you mean by the build fails?

**Comment by LordReigns** [comment]
In the repository, it shows the Travis build fails right? @ankushduacodes

**Comment by ankushduacodes** [comment]
I am not sure why this us the case, if you look into [this](https://travis-ci.org/github/mozilla/OpenWPM/builds/744644083)
It shows that all the tests have passed, @vringar Any idea whats going on?

**Comment by LordReigns** [comment]
> Update 2: I just ran the test on my local machine and can confirm that it fails, I will try to debug it in the mean time

Cool. It says test passed in my machine though.  Thanks @ankushduacodes .

**Comment by ankushduacodes** [comment]
@LordReigns There have been some commits and there was a new release just published few hours ago, Have you rebased your forked repo with mozilla/OpenWPM master

**Comment by LordReigns** [comment]
Oh I haven't, really sorry about that @ankushduacodes

**Comment by ankushduacodes** [comment]
Update 3: So in the latest release something changed(I am not sure what), that triggered a change in how commands in the test database are stored: In version v0.12.0, The commands were stored by the convention where project name was attached to command in such manner ` <class 'OpenWPM.openwpm.Commands.Types.GetCommand'>` but in the latest release v0.13.0, the commands are stored in this manner ` <class 'openwpm.Commands.Types.GetCommand'>`

I am not sure what sparked the changed to occur, and can confirm that test_timer.py also fails due to this issue on my local machine, although I am not sure why it didnt fail in travis CI.

**Comment by LordReigns** [comment]
Can you open an issue, so we can discuss this separately? @ankushduacodes

**Comment by vringar** [comment]
When releasing v0.13.0 I updated our version of pytest so maybe that changed something.
Please file a PR that changes the where clause to use the new name

**Comment by ankushduacodes** [comment]
@vringar Do you want me to file an issue for this? and then make a pull request or should I just make pull request fixing it?

**Comment by LordReigns** [comment]
> Please file a PR that changes the where clause to use the new name

I have updated my local repo. I also have submitted a new PR. Hopefully it will work well this time.

Thank you

**Pull request merged**